src/main/java/com/smartcar/sdk/ApiClient.java

```diff
@@ -80,7 +80,7 @@ protected static Response execute(Request request) throws SmartcarException {
     String unitHeader = response.header("sc-unit-system");
 
     String ageHeader = response.header("sc-data-age");
-    DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+    DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
Sounds like `SimpleDateFormat` actually does not parse ISO 8601 compliant strings: https://stackoverflow.com/a/2202300/1047268

There's a couple solutions on that thread given that we are targeting JDK 1.7. If we limit ourselves to JDK 1.8, we get access to [LocalDateTime.parse()](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#parse-java.lang.CharSequence-) built in. If we want to use a robust solution, we could always import in [Joda-Time](http://www.joda.org/joda-time/).